### ISSUE TITLE
Grafana UI: V9 beta packages cause plugin tests to fail

### ISSUE THREAD

**Issue by mikhail-vl** [grafana/grafana#50363]
**What happened**:

Can't build custom data sources and application plugins using Grafana 9.0.0-beta3

```
yarn run v1.22.17
$ grafana-toolkit plugin:build --coverage
  Using Node.js v16.15.0
  Using @grafana/toolkit v9.0.0-beta3
✔ Preparing
✔ Linting
 FAIL  src/components/RootPage/RootPage.test.tsx
  ● Test suite failed to run

    Cannot find module 'indexof' from 'node_modules/@grafana/ui/index.development.js'

    Require stack:
      node_modules/@grafana/ui/index.development.js
      node_modules/@grafana/ui/index.js
      src/components/RootPage/RootPage.test.tsx

      at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (node_modules/@grafana/ui/index.development.js:11:20)

 FAIL  src/components/Config/Config.test.tsx
  ● Test suite failed to run

    Cannot find module 'indexof' from 'node_modules/@grafana/ui/index.development.js'

    Require stack:
      node_modules/@grafana/ui/index.development.js
      node_modules/@grafana/ui/index.js
      node_modules/@grafana/runtime/index.development.js
      node_modules/@grafana/runtime/index.js
      src/components/Config/Config.tsx
      src/components/Config/Config.test.tsx

      at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (node_modules/@grafana/ui/index.development.js:11:20)

 FAIL  src/module.test.ts
  ● Test suite failed to run

    Cannot find module 'indexof' from 'node_modules/@grafana/ui/index.development.js'

    Require stack:
      node_modules/@grafana/ui/index.development.js
      node_modules/@grafana/ui/index.js
      node_modules/@grafana/runtime/index.development.js
      node_modules/@grafana/runtime/index.js
      src/components/Config/Config.tsx
      src/components/Config/index.ts
      src/components/index.ts
      src/module.ts
      src/module.test.ts

      at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (node_modules/@grafana/ui/index.development.js:11:20)
```

**What you expected to happen**:

Build successful.

**How to reproduce it (as minimally and precisely as possible)**:

1. Download https://github.com/VolkovLabs/volkovlabs-abc-datasource
2. Remove `indexof` from `package.json`, `yarn install`
3. Try to build

**Anything else we need to know?**:

Adding `indexof` to the `package.json` resolves the issue.

**Environment**:
- Grafana version: 9.0.0-beta3
- Data source type & version: Custom Template
- OS Grafana is installed on: 9.0.0-beta3
- User OS & Browser: N/A
- Grafana plugins:
- Others:

**Comment by jackw** [comment]
Thanks so much for reporting this issue @mikhail-vl! 🥇 

I've reproduced it locally. I originally thought it was related to the changes to toolkit however it's actually a problem with a transitive dependency in `@grafana/ui`. I'm still tracing the cause (most likely a version bump to a package somewhere in the past few weeks) but for now it appears all `@grafana/ui@v9.0.0-betaX` releases have this issue.

**Comment by mikhail-vl** [comment]
@jackw Thank you for confirming. I hope it's going to be resolved in 9.0.

**Comment by jackw** [comment]
@mikhail-vl I've added a fix to `@grafana/ui` for this issue which will arrive in v9.0.0. 

I did the following from a fresh clone of your [volkovlabs-abc-datasource](https://github.com/VolkovLabs/volkovlabs-abc-datasource) repo and tests now pass and project successfully builds.

1. remove `indexof` from `package.json`
2. replace `"@grafana/ui": "9.0.0-beta3"` with `"@grafana/ui": "9.0.0-66256pre"` in `package.json`
3. run `yarn && yarn build`
